media-gfx/netgen/metadata.xml

```diff
@@ -5,8 +5,17 @@
 		<email>sci-mathematics@gentoo.org</email>
 		<name>Gentoo Mathematics Project</name>
 	</maintainer>
+	<maintainer type="person" proxied="yes">
+		<email>waebbl-gentoo@posteo.net</email>
+		<name>Bernd Waibel</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
```

Comment:
Please drop proxy-maint here, they don't maintain packages in the science overlay.

Reply:
Adding proxy-maint and myself was in preparation for the move to ::gentoo.
Still remove it?

Reply:
> Adding proxy-maint and myself was in preparation for the move to ::gentoo. Still remove it?

For adding it here we're going to have to remove it. When it is moved we can re-add it.